src/core/qgsfeaturerequest.h

```diff
@@ -218,47 +218,47 @@ class CORE_EXPORT QgsFeatureRequest
      *
      * \since QGIS 2.14
      */
-    class OrderBy : public QList<QgsFeatureRequest::OrderByClause>
+    class CORE_EXPORT OrderBy : public QList<QgsFeatureRequest::OrderByClause>
```

Comment:
does this build with MSVC?

Reply:
@jef-n I unfortunately dont have a MSVC toolchain to test this.

Reply:
I've given this a test, and unfortunately it does break the msvc build. Does mingw build if you leave individual member CORE_EXPORTs, but insert this new class CORE_EXPORT? If so we could just #ifdef two different declarations for the class. If not, I can't think of any other way to resolve this.